navigation/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/SummaryBottomSheet.java

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.services.android.navigation.ui.v5.summary;
+
+public class SummaryBottomSheet {
```

Comment:
What is this class?

Reply:
Was a placeholder for the project structure -- all code was added in https://github.com/mapbox/mapbox-navigation-android/pull/185